frontends/web/src/components/bluetooth/bluetooth.tsx

```diff
@@ -31,6 +34,31 @@ const isConnectedOrConnecting = (peripheral: TPeripheral) => {
 const _Bluetooth = () => {
   const { t } = useTranslation();
   const state = useSync(getState, syncState);
+  const [showConnectionIssues, setShowConnectionIssues] = useState(false);
+  const [dialogOpen, setDialogOpen] = useState(false);
+  const scanningTimeoutRef = useRef<ReturnType<typeof setTimeout> | null>(null);
+  const TIMEOUT_MS = 30000;
+
+  useEffect(() => {
+    if (state?.scanning) {
```

Comment:
If it's scanning and there are peripherals, but the user has not clicked on any yet, there are no connection issues, user just hasn't tapped on any yet.